gst/nnstreamer/nnstreamer_conf.c

```diff
@@ -34,6 +34,12 @@ static const gchar *subplugin_prefixes[] = {
   [NNSCONF_PATH_END] = NULL
 };
 
+#ifdef G_OS_WIN32
+static const gchar root_path_prefix[] = "c:\\";
```

Comment:
how about moving these block (root_path_prefix) into the loop?
actually root_path_prefix is used only in the loop (line 278), so remove static and move into the loop.

Reply:
@jaeyun-jung I modified my patch as you commented. Thanks.